app/src/main/java/com/example/mobiledger/common/utils/DummyNavigator.kt

```diff
@@ -1,3 +1,5 @@
 package com.example.mobiledger.common.utils
 
```

Comment:
Remove Dummy Navigator. We don't need it, we'll use BaseNavigator in classes where navigator is not required.